com.openwall.John.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Version=1.9.0J1
+Type=Application
+Terminal=1
+Name=John the Ripper Community Edition
+GenericName=John the Ripper Jumbo
+Comment=John the Ripper "Jumbo" password cracker
+TryExec=john
```

Comment:
Should be upstream.

Reply:
I'm not actually if it's good idea to provide desktop launcher if application doesn't provide TUI.